website/chatui/Controllers/ChatController.cs

```diff
@@ -19,41 +18,45 @@ public class ChatController(
     private readonly IOptionsMonitor<ChatApiOptions> _options = options;
     private readonly ILogger<ChatController> _logger = logger;
 
-    [HttpPost]
-    public async Task<IActionResult> Completions([FromBody] string prompt)
+    [HttpPost("{threadId}")]
+    public async Task<IActionResult> Completions([FromRoute] string threadId, [FromBody] string prompt)
     {
         if (string.IsNullOrWhiteSpace(prompt))

```

Comment:
Can you add a TODO related to security.  Since you are now allowing the client to freely step on, and into any random thread.  You don't need to solve the security issue, just need to call it out.

Reply:
good catch @ckittel 

done | TODO added from f5c39fe5808ac53b39d4e4a08a1e6c9e3c07746f